lib/commands/setup.ts

```diff
@@ -0,0 +1,29 @@
+export class SetupCommand implements ICommand {
+	public allowedParameters: ICommandParameter[] = [];
+
+	constructor(private $doctorService: IDoctorService) { }
+
+	public async execute(args: string[]): Promise<any> {
+		return this.$doctorService.runSetupScript();
+	}
+
+	public async canExecute(args: string[]): Promise<boolean> {
+		return true;
+	}
+}
+$injector.registerCommand("setup", SetupCommand);
+
+export class CloudSetupCommand implements ICommand {
+	public allowedParameters: ICommandParameter[] = [];
+
+	constructor(private $extensibilityService: IExtensibilityService) { }
+
+	public async execute(args: string[]): Promise<any> {
+		return this.$extensibilityService.installExtension("nativescript-cloud");
```

Comment:
Should check if extension is already installed. If this is the case it shouldn't install again.